go.mod

```diff
@@ -39,7 +39,7 @@ require (
 	github.com/oapi-codegen/runtime v1.1.1
 	github.com/operator-framework/api v0.27.0
 	github.com/operator-framework/operator-lifecycle-manager v0.27.0
-	github.com/percona/everest-operator v0.6.0-dev1.0.20250326082629-7c6a6a61ada3
+	github.com/percona/everest-operator v0.6.0-dev1.0.20250326194616-2c787fc12baa
```

Comment:
Update to the correct version once https://github.com/percona/everest-operator/pull/718 is merged to main.